### PR TITLE
Added the join ride dialog box

### PR DESCRIPTION
**Pull request by DarNachmani**
https://github.com/CommuneCar/mono-app/assets/62894661/4f38efa2-000f-4f75-92b3-17821ce5eeef

### CODE REVIEW

**Comment by DarNachmani** [comment]
@Kovler10 
Don't have the BE request yet - waiting to see in the meeting today the convention of the implementation of the other requests

File: apps/ui/src/Components/JoinRide/JoinRide.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogTitle from '@mui/material/DialogTitle';
+import Button from '@mui/material/Button';
+import Checkbox from '@mui/material/Checkbox';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import TextField from '@mui/material/TextField';
```
Comment (Kovler10) [inline apps/ui/src/Components/JoinRide/JoinRide.tsx:9]: please put all imports under one @mui/material import

File: apps/ui/src/Components/JoinRide/JoinRide.tsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+import { Grid } from '@mui/material';
+
+interface IJoinRideDialog {
```
Comment (Kovler10) [inline apps/ui/src/Components/JoinRide/JoinRide.tsx:12]: joinRideProps

File: apps/ui/src/Components/JoinRide/JoinRide.tsx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  setOpen: (isOpen: boolean) => void;
+  isOpen: boolean;
+}
+
+function JoinRideDialog({ isOpen, setOpen }: IJoinRideDialog) {
```
Comment (Kovler10) [inline apps/ui/src/Components/JoinRide/JoinRide.tsx:17]: `const JoinRide: React.FC<JoinRideProps> = ({ isOpen, setIsOpen }) => { `

File: apps/ui/src/Components/JoinRide/JoinRide.tsx [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+  const [isChecked, setChecked] = useState(false);
+  const [number, setNumber] = useState(1);
```
Comment (Kovler10) [inline apps/ui/src/Components/JoinRide/JoinRide.tsx:19]: number of what?

File: apps/ui/src/Components/JoinRide/JoinRide.tsx [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+          }}
+        >
+          Join
+        </Button>
+      </DialogActions>
+    </Dialog>
+  );
+}
+
+export default JoinRideDialog;
```
Comment (Kovler10) [inline apps/ui/src/Components/JoinRide/JoinRide.tsx:93]: avoid using export default

**Review by Kovler10 — CHANGES_REQUESTED** [review]
Nice, please remove all js build files

**Review by Kovler10 — APPROVED** [review]

**Pull request merged**